layers/+themes/dark-window/README.org

```diff
@@ -0,0 +1,26 @@
+#+TITLE: dark-window layer
+
+* Table of Contents                                        :TOC_4_gh:noexport:
+ - [[#description][Description]]
+ - [[#install][Install]]
+ - [[#screenshots][Screenshots]]
+
+* Description
+Gnome 3's default theme Adwaita has support for both light and dark window
+decorations. Many media apps like Totem use the dark theme. It puts the focus
+on the content and it's easier on the eyes.
+
+Emacs with a dark theme and the large white-ish window decoration isn't really a
+fit on my desktop, but that can be easily fixed using a bit of elisp with xprop!
```

Comment:
This is a direct quote from the article, so it should be formatted in a quote block. Don't plagiarize.